frontend/src/v5/ui/controls/inputs/checkbox/checkbox.component.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { FormInputProps } from '@controls/inputs/inputController.component';
+import { FormControlLabel, FormControlLabelProps, Checkbox as MuiCheckbox } from '@mui/material';
+
+export type CheckboxProps = FormInputProps & FormControlLabelProps & {
+	onClick?: (event) => any,
+};
+
+export const Checkbox = ({ value, onClick, ...props }: CheckboxProps) => (
+	<FormControlLabel
+		control={<MuiCheckbox checked={value} onClick={onClick} />}
```

Comment:
Yes, it works. (By the time you reviewed this PR) Most of the event handlers could be passed and they were correctly triggered, with the exception of `onChange` which would have still been triggered, but would have overridden the default onchange behaviour of a controlled element preventing its value update (the same would have been true also for FormTextField). I fixed this in the `InputController` component.

Also, the fact that the `onClick` is extracted from the other props, is simply because if it passed to `FormControlLabel`, it is triggered twice on component click, so it has to be passed directly to the `MuiCheckbox`